### PR TITLE
fix the script variable rename bugs in state%pdel and sflxnam

### PR DESCRIPTION
**Pull request by xyuan**
This is the issue caused by the script, it replace the state%pdel to state%dpres, and sflxnam to precip_ice_fluxnam. These two variables shouldn't be replaced.

BFB

### CODE REVIEW

File: components/cam/src/physics/cam/micro_p3_interface.F90 [hunk 1]
```diff
@@ -22,7 +22,7 @@ module micro_p3_interface
                             physics_ptend_init
   use physconst,      only: mwdry, cpair, mwh2o, gravit, rair, cpliq, pi, &
                             rh2o, latvap, latice, tmelt, rhoh2o, rairv 
-  use constituents,   only: cnst_add, pcnst, precip_ice_fluxnam, apcnst, bpcnst, pcnst,&
+  use constituents,   only: cnst_add, pcnst, sflxnam, apcnst, bpcnst, pcnst,&
```
Comment (AaronDonahue) [inline components/cam/src/physics/cam/micro_p3_interface.F90:25]: if you wanted to keep `precip_ice_fluxnam` for other reasons you could use something like `precip_ice_fluxnam->sflxnam` here.

**Comment by xyuan** [comment]
No, this is not my intention, it is the script that replace all the string in this file, unfortunately this one contains same chars.

**Comment by AaronDonahue** [comment]
per keeping the `precip_ice_fluxnam`.  It looks like it would only need to be changed back in a few places.  So you wouldn't have to run a script.  You probably just find/replace manually.  I just recall that the original PR was meant to adjust the names of variables to be more clearly understood.  So if `precip_ice_fluxnam` is a part of that spirit, we could still keep it in `micro_p3_interface` but using the ability to load a public variable from a module yet assign it a local name.  All you need is the `->` syntax.  We use it often in other parts of the EAM code, so it shouldn't be outside the standard coding practice.

**Comment by xyuan** [comment]
Please don’t approve it, the test passed doesn’t mean it works with full SCREAM run.

**Comment by xyuan** [comment]
Yes, I rerun the script on this file, and manually rename the precip_ice_fluxnam back to it’s origin name, there are too many arguments for p3_main.

**Comment by xyuan** [comment]
@AaronDonahue 
This all fixed, see below:
[yuanx@login5.summit scream_rename]$ cime/scripts/create_test ERS.ne4pg2_ne4pg2.F2010-SCREAM-LR.summit_gnu
No project info available
create_test will do up to 1 tasks simultaneously
create_test will use up to 105 cores simultaneously
Creating test directory /gpfs/alpine/cli115/proj-shared/yuanx/e3sm_scratch/ERS.ne4pg2_ne4pg2.F2010-SCREAM-LR.summit_gnu.20200819_174703_f4gy8c
RUNNING TESTS:
  ERS.ne4pg2_ne4pg2.F2010-SCREAM-LR.summit_gnu
Starting CREATE_NEWCASE for test ERS.ne4pg2_ne4pg2.F2010-SCREAM-LR.summit_gnu with 1 procs
Finished CREATE_NEWCASE for test ERS.ne4pg2_ne4pg2.F2010-SCREAM-LR.summit_gnu in 5.096089 seconds (PASS)
Starting XML for test ERS.ne4pg2_ne4pg2.F2010-SCREAM-LR.summit_gnu with 1 procs
Finished XML for test ERS.ne4pg2_ne4pg2.F2010-SCREAM-LR.summit_gnu in 0.352402 seconds (PASS)
Starting SETUP for test ERS.ne4pg2_ne4pg2.F2010-SCREAM-LR.summit_gnu with 1 procs
Finished SETUP for test ERS.ne4pg2_ne4pg2.F2010-SCREAM-LR.summit_gnu in 4.333197 seconds (PASS)
Starting SHAREDLIB_BUILD for test ERS.ne4pg2_ne4pg2.F2010-SCREAM-LR.summit_gnu with 1 procs
Finished SHAREDLIB_BUILD for test ERS.ne4pg2_ne4pg2.F2010-SCREAM-LR.summit_gnu in 105.598312 seconds (PASS)
Starting MODEL_BUILD for test ERS.ne4pg2_ne4pg2.F2010-SCREAM-LR.summit_gnu with 6 procs
Finished MODEL_BUILD for test ERS.ne4pg2_ne4pg2.F2010-SCREAM-LR.summit_gnu in 178.710354 seconds (PASS)
Starting RUN for test ERS.ne4pg2_ne4pg2.F2010-SCREAM-LR.summit_gnu with 1 proc on interactive node and 96 procs on compute nodes
Finished RUN for test ERS.ne4pg2_ne4pg2.F2010-SCREAM-LR.summit_gnu in 4.573727 seconds (FAIL). [COMPLETED 1 of 1]

The run failed because it asked for 03:00 hours, manually restart the job and it runs to the end. (see the output below)

2: 0:   sysmem size=2028.3 MB rss=883.5 MB share=425.9 MB text=22.9 MB datastack=0.0 MB
2: 0:   sysmem size=2028.3 MB rss=883.5 MB share=425.9 MB text=22.9 MB datastack=0.0 MB
2: 0:   sysmem size=2028.3 MB rss=883.5 MB share=425.9 MB text=22.9 MB datastack=0.0 MB

**Pull request merged**